### PR TITLE
Throw errors from requiring messages

### PR DESCRIPTION
**Pull request by chris-smith**
-If an error occurs while requiring a message package, throw it.
-Add tests for expected messages, throwing error when message doesn't exist

Relates to #93, #85

### CODE REVIEW

File: src/utils/message_utils.js [hunk 1]
```diff
@@ -170,12 +170,7 @@ let MessageUtils = {
   },
 
   loadMessagePackage(msgPackage) {
-    try {
-      messagePackageMap[msgPackage] = ros_msg_utils.Find(msgPackage);
-    }
-    catch (err) {
-      throw new Error(`Unable to include message package ${msgPackage} - ${err}`);
```
Comment (IanTheEngineer) [inline src/utils/message_utils.js:177]: Wasn't this already re-throwing an error? Is the issue that it was swallowing the original error message?

**Review by IanTheEngineer — APPROVED** [review]
Looks good for merging!

File: src/utils/message_utils.js (hunk 1 above)
Comment (chris-smith) [reply src/utils/message_utils.js:177]: The other function was swallowing the error but this was obscuring the actual error by changing the stack, making it harder for people to debug issues when they occur.

Comment (IanTheEngineer) [reply src/utils/message_utils.js:177]: :+1:

**Pull request merged**